pkg/system/phase2_creating.go

```diff
@@ -194,6 +196,31 @@ func (r *Reconciler) SetDesiredServiceDBForPostgres() error {
 	return nil
 }
 
+func (r *Reconciler) setPostgresUserPass(c *corev1.Container) {
+	for j := range c.Env {
+		switch c.Env[j].Name {
+		case "POSTGRESQL_USER":
+			c.Env[j].ValueFrom = &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: r.SecretDB.Name,
```

Comment:
Is there a reason we need to reconcile these values??
Why not just put those in the yaml itself like:
```yaml
env:
  - name: POSTGRESQL_USER
     valueFrom:
       secretKeyRef:
         name: noobaa-db
         key: user
  - name: POSTGRESQL_PASSWORD
     valueFrom:
       secretKeyRef:
         name: noobaa-db
         key: password
```

Reply:
@guymguym sounds good

Reply:
what about the upgrade, don't we need to add the new init container?